From 8a5701854a6009dcf69f7b6b22b127fbfec3fc4d Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@nxp.com>
Date: Fri, 24 Nov 2017 17:15:04 +0800
Subject: [PATCH 3009/5242] MLK-16976-1 ARM64: dts: fsl-imx8: add OTG register
 regions for Cadence USB3

commit  acf19c646b647ad7e154e5061fb7541480d85bff from
https://source.codeaurora.org/external/imx/linux-imx.git

Add OTG register regions for Cadence USB3.

BuildInfo:
- SCFW 245582b, IMX-MKIMAGE 0ad6069a, ATF 6bd98a3
- U-Boot 2017.03-imx_v2017.03+gfa65b0a

Acked-by: Jun Li <jun.li@nxp.com>
Signed-off-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi  |    3 ++-
 arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi |    3 ++-
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
index 9819f00..b09f2b7 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
@@ -2682,7 +2682,8 @@
 		reg = <0x0 0x5B110000 0x0 0x10000>,
 			<0x0 0x5B130000 0x0 0x10000>,
 			<0x0 0x5B140000 0x0 0x10000>,
-			<0x0 0x5B160000 0x0 0x40000>;
+			<0x0 0x5B160000 0x0 0x40000>,
+			<0x0 0x5B120000 0x0 0x10000>;
 		interrupt-parent = <&gic>;
 		interrupts = <GIC_SPI 271 IRQ_TYPE_LEVEL_HIGH>;
 		clocks = <&clk IMX8QM_USB3_LPM_CLK>,
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
index fa0bbdf..b07a12c 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
@@ -1620,7 +1620,8 @@
 		reg = <0x0 0x5B110000 0x0 0x10000>,
 			<0x0 0x5B130000 0x0 0x10000>,
 			<0x0 0x5B140000 0x0 0x10000>,
-			<0x0 0x5B160000 0x0 0x40000>;
+			<0x0 0x5B160000 0x0 0x40000>,
+			<0x0 0x5B120000 0x0 0x10000>;
 		interrupt-parent = <&gic>;
 		interrupts = <GIC_SPI 271 IRQ_TYPE_LEVEL_HIGH>;
 		clocks = <&clk IMX8QXP_USB3_LPM_CLK>,
-- 
1.7.9.5

